admin/templates/configuration/bin/qserv-stop.sh

```diff
@@ -1,16 +1,54 @@
 #!/bin/sh
 
+# LSST Data Management System
+# Copyright 2015 LSST Corporation.
```

Comment:
This should probably be 20xx-2015 (find the year when this file was created from git)